clang/www/c_status.html

```diff
@@ -191,7 +191,7 @@ <h2 id="c2y">C2y implementation status</h2>
     <tr>
       <td>Slay Some Earthly Demons III</td>
       <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3341.pdf">N3341</a></td>
-      <td class="unknown" align="center">Unknown</td>
+      <td class="full" align="center">Yes</td>
```

Comment:
I wonder why the patch comes with release note, yet the official support is not claimed starting from clang 20?

Reply:
It went from UB to implementation-defined, but we've always defined (and failed to document) the behavior. So there were no substantive changes in Clang 20, which is why I claimed "Yes" instead of Clang 20. The release note is mostly for user awareness about the change in diagnostic behavior.